airflow/utils/scheduler_health.py

```diff
@@ -35,12 +37,12 @@ def do_GET(self):
         if self.path == "/health":
             try:
                 with create_session() as session:
-                    scheduler_job = (
-                        session.query(Job)
+                    scheduler_job = session.scalars(
```

Comment:
```suggestion
                    scheduler_job = session.scalar(
```

The original query uses `first()`; changing it to `scalars` is wrong (I believe)

Reply:
fixed